meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,121 @@
+# Wayfair plugin: Initial Setup
+
+## Prerequisites
+* [Wayfair API credentials](obtaining_credentials.md).
+
+* Administrative rights on the Plentymarkets system
+
+* [Installation](plugin_installation.md) of the Wayfair plugin - [view release notes](https://github.com/wayfair-contribs/plentymarkets-plugin/releases)
+
+
+## 1. Authorizing the Wayfair Plugin to access Wayfair interfaces
+After the plugin is installed in your Plentymarkets Plugin Set, the plugin must be configured to use the correct credentials when connecting to Wayfair's interfaces.
+
+* **The authorization procedure must be performed for any Plugin Set that contains the Wayfair plugin**.
+* Copying a Plugin Set will copy the authorization information to the new plugin set.
+* An exported or imported Plugin set may include the authorization information.
+
+The authorization steps are as follows:
+1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`
+
+2. Locate the Plugin Set that is linked to the client with which Wayfair will be used.
+
+3. Click on the `Edit` button for the desired Plugin set
+
+4. In the Wayfair row of the Plugin set, click on the `Settings` button.

```

Comment:
Is there ever more than one row?

Reply:
Plugin Sets currently only can have one row per plugin (regardless of version), so if there's more than one Wayfair row, something has gone very wrong.